pkg/kubelet/volume/empty_dir/empty_dir.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
puke, really?  Even for dirs with underscores in their names?  Do we have to follow this?  I find that  sort of obfuscatory